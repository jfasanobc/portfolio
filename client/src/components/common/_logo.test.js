import React from "react";
import renderer, { create } from "react-test-renderer";
import Logo from "./logo";

describe("Logo appears on page as a link to root uri? ", () => {
  test("it has rendered and is a link to /", () => {
    const logo = create(<Logo />);
    expect(logo.toJSON()).toMatchSnapshot();
  });
});
