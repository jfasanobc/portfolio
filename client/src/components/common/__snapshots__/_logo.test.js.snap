// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo appears on page as a link to root uri?  it has rendered and is a link to / 1`] = `
<a
  href="/"
>
  <img
    className="logo"
    src="./assets/img/logo.png"
  />
</a>
`;
